Remotion/Data/DomainObjects/Persistence/Rdbms/IRdbmsProviderCommandWithReadOnlySupport.cs

```diff
@@ -0,0 +1,27 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+
+namespace Remotion.Data.DomainObjects.Persistence.Rdbms;
+
+/// <summary>
+/// Represents a command that does to modify the database and returns a value, to be executed by a storage provider.
```

Comment:
```suggestion
/// Represents a command that will not modify the database, to be executed by a storage provider.
```
@martin-herwich I believe this is the one that will not modify?

Reply:
Correct, it should have been "does not" instead of "does to".